dtos/provisionwatcher.go

```diff
@@ -18,26 +18,20 @@ type ProvisionWatcher struct {
 	Labels              []string            `json:"labels,omitempty" yaml:"labels,omitempty"`
 	Identifiers         map[string]string   `json:"identifiers" yaml:"identifiers" validate:"gt=0,dive,keys,required,endkeys,required"`
 	BlockingIdentifiers map[string][]string `json:"blockingIdentifiers,omitempty" yaml:"blockingIdentifiers,omitempty"`
-	ProfileName         string              `json:"profileName" yaml:"profileName" validate:"required,edgex-dto-none-empty-string,edgex-dto-rfc3986-unreserved-chars"`
-	ServiceName         string              `json:"serviceName" yaml:"serviceName" validate:"required,edgex-dto-none-empty-string,edgex-dto-rfc3986-unreserved-chars"`
 	AdminState          string              `json:"adminState" yaml:"adminState" validate:"oneof='LOCKED' 'UNLOCKED'"`
-	AutoEvents          []AutoEvent         `json:"autoEvents,omitempty" yaml:"autoEvents,omitempty" validate:"dive"`
-	Properties          map[string]any      `json:"properties,omitempty" yaml:"properties,omitempty"`
+	DiscoveredDevice    DiscoveredDevice    `json:"discoveredDevice" yaml:"deviceFields" validate:"dive"`
```

Comment:
Why `deviceFields` in YAML and `discoveredDevice` in JSON?

Reply:
updated